core-api/pyproject.toml

```diff
@@ -35,6 +35,8 @@ litellm = "^1.41.11"
 pytest = "^8.2.2"
 pytest-cov = "^5.0.0"
 pytest-asyncio = "^0.23.6"
+pytest-dotenv = "^0.5.2"
+mypy = "^1.9.0"
```

Comment:
not a blocker but i dont think mypy belongs here

Reply:
good point, missed this in a merge from main